### ISSUE TITLE
[ BUG ] `Find-FalconHostname` returns maximum of 100 results

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#369]
**Describe the bug**
`Find-FalconHostname` cuts off at the first 100 results instead of returning the entire list

**To Reproduce**
Add more than 100 hostnames to a text file and use it with `Path`, or submit an array of more than 100 values.

**Environment (please complete the following information):**
 - OS: Windows 11
 - PowerShell: 5.1
 - PSFalcon: 2.2.5

**Comment by bk-cs** [comment]
I was able to reproduce this and determine that the problem comes from the reuse of a variable during the `for` loop. This mistake ruins the list of supplied hostname values after the first request (which contains 100 hostnames) is submitted, which ultimately limits the output to 100 results.

I'm working on a fix to be implemented in the 2.2.7 release and checking if there are any performance improvements to be made with the command in general.

**Comment by bk-cs** [comment]
If you'd like to fix your local `Find-FalconHostname` command before the next release and you're currently using version 2.2.6, use the following commands:
Update file using PowerShell 5.1 on Windows:
```powershell
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/CrowdStrike/psfalcon/aeba5e11d56fccf684b2a609cf28f8f0e91c08bf/public/psf-devices.ps1 -UseBasicParsing).Content > $HOME\Documents\WindowsPowerShell\Modules\PSFalcon\2.2.6\Public\psf-devices.ps1
```
Update file using PowerShell Core on Windows:
```powershell
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/CrowdStrike/psfalcon/aeba5e11d56fccf684b2a609cf28f8f0e91c08bf/public/psf-devices.ps1 -UseBasicParsing).Content > $HOME\Documents\PowerShell\Modules\PSFalcon\2.2.6\Public\psf-devices.ps1
```
Update file using PowerShell Core on Linux/Mac:
```powershell
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/CrowdStrike/psfalcon/aeba5e11d56fccf684b2a609cf28f8f0e91c08bf/public/psf-devices.ps1 -UseBasicParsing).Content > $HOME/.local/share/PowerShell/Modules/PSFalcon/2.2.6/Public/psf-devices.ps1
```
If you're using an earlier version, replace `2.2.6` with your currently installed version number.

**Comment by bk-cs** [comment]
This fix is included in the [2.2.7 release](https://github.com/CrowdStrike/psfalcon/releases/tag/2.2.7).
